docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -28,17 +28,267 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+
+1. Set the following controls on the Generation Example GSE VI and leave all other values at their defaults.
+    > Note: VI controls and indicators can be reset to default values by clicking on the **Edit** menu and selecting the **Reinitialize Values to Default** option.
+
+    | Tab | Setting | Value |
+    |---|---|---|
+    | Resource | RIO Device | [System Specific] |
+    | Resource | Bitfile Path | [Refer to Bitfile Path in the PXIe-148X Generation GSE Help](../../reference/gettingstartedexample/gse-gen-help.md#table-of-pxie-148x-generation-bitfiles) |
+    | Resource | Display Generated Images | Disabled |
+    | Resource | Log GPIO to Disk | Enabled |
+
+### Creating a TDMS File for the GPIO Tutorial
+
+1. Open the Create CSI-2 Packet TDMS Files utility in the LabVIEW project.
+
+2. Make the following modifications on the Create CSI-2 Packet TDMS Files utility front panel.
+    - Set the **Number of Frames** control to <font face = "courier new">60</font>
+    - Update the **Frame Data Configuration** cluster with the following settings.
+        - Set the **horizontal resolution** control value to <font face = "courier new">640</font>.
+        - Set the **vertical resolution** control value to <font face = "courier new">480</font>.
+    - In the **Frame Timing Configuration** cluster, set the **desired frame rate (fps)** control to <font face = "courier new">1</font>.
+    
+    > The Create TDMS Files utility settings are now similar to the image below.
+
+    ![Create TDMS Files Settings for the GPIO Turtorial](../../images/PXIe-148X-CreateTDMS-GenGPIO.png)
+
+3. Run the VI to generate a TDMS file for channel SI0 containing 60 frames at 640x480 resolution.
+    > The frame rate of 1 frame per second will cause the generation to run for 1 minute, which will allow enough time to manually read and write to GPIO during the generation.
+
+### Manually Reading and Writing to GPIO
+
+This tutorial shows you how to perform manual reads and writes to GPIO during generation and display digital waveforms for the resulting GPIO timestamp data.
+> Note: The [Creating a TDMS File for the GPIO Tutorial](#creating-a-tdms-file-for-the-gpio-tutorial) section should be run prior to running the steps in this tutorial.
+
+1. Select the **GPIO** tab and make the following modifications.
+    - Update the **GPIO to Display** array with the following settings.
+        - At index 0 select **Ser0 GPIO** in the **GPIO Bank** control and set the **GPIO Number** to <font face = "courier new">0</font>. 
+
+        > Setting these values enables display of GPIO traffic for GPIO line 0 on the SO0 channel. Optionally add any additional GPIO lines to display.
+        >
+        > The **GPIO to Display** control specifies GPIO lines to display on the GPIO Timestamps Waveform after the acquisition completes. Timestamps for GPIO lines not included in the **GPIO to Display** array are logged, but not displayed.
```

Comment:
update to: "after the generation completes"

Reply:
Done.